Podfile

```
@@ -76,6 +76,9 @@ end
 
 def gutenberg(options)
     options[:git] = 'http://github.com/wordpress-mobile/gutenberg-mobile/'
+    if ENV['LOCAL_GUTENBERG']
+      options = { :path => ENV['LOCAL_GUTENBERG'] }
```

We could use a default path here, since `gutenberg-mobile` is likely to be in the same directory as WPiOS.

```suggestion
      options = { :path => ENV['GUTENBERG_PATH'] || '../gutenberg-mobile' }
```

---

If we use a default local path, won't it _always_ use gutenberg from local?

---

Nope, because we still have the `if ENV['LOCAL_GUTENBERG']` check above, but it's treated as a boolean rather than a path.

---

Right! Sounds good :)